dask/array/numpy_compat.py

```diff
@@ -1,14 +1,15 @@
 import warnings
-from distutils.version import LooseVersion
 
 import numpy as np
+from packaging.version import parse as parse_version
```

Comment:
Nitpick: It might be easier for readers to reason about if we instead `import packaging.version` and then call `packaging.version.parse(...)` directly when parsing a version string. Otherwise, folks will often need to look up where `parse_version` is coming from. Additionally, it's often difficult to use import aliases uniformly across a project.

Reply:
This makes many lines quite long, and `black` formats them quite poorly, so the shorter name seemed better. Also, it matches the name used in `setuptools`, in case you wanted to fall back to that when `packaging` is unavailable (it used to have its own implementation.)